questionnaires/vaping_weekly_survey/vaping_weekly_survey_armt.json

```diff
@@ -0,0 +1,856 @@
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 I have no phlegm (mucus) on my chest at all "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 My chest is full of phlegm (mucus)"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "phlegm_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "chest",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item below, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 My chest does not feel tight at all "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 My chest feels very tight"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "chest_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "breath",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item below, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 When I walk up a hill or a flight of stairs I am not out of breath "
```

Comment:
Same here

Reply:
Not sure I can easily think of a shorter version.